src/pages/projects.js

```diff
@@ -1,11 +1,18 @@
+import { PageHeader } from '../components/page-header/page-header'
+import { useState } from 'react'
 import { ProjectContent } from '../components/project-content/project-content'
+import { projectsDatas } from '../components/projects-datas'
 export const ProjectsPage = () => {
+    const [datas, setDatas] = useState([{ projectsDatas }])
```

Comment:
this one hasn't updated please update it as follows
`const [datas, setDatas] = useState(ProjectsDatas)`